.github/workflows/drc.yml

```diff
@@ -20,7 +20,7 @@ name: DRC
 on: 
     push:
     pull_request:
-        types: [opened, reopened]
+        branches: [ main]
```

Comment:
Changed this line to also trigger on force-pushes.

Reply:
@PhilippNauck can you please take a look at this ?